spec/requests/posts_spec.rb

```diff
@@ -23,16 +23,16 @@
       get '/users/1/posts/1'
     end
 
-    it 'returns code 200' do
-      expect(response).to have_http_status(:success)
-    end
+    # it 'returns code 200' do
+    #   expect(response).to have_http_status(:success)
+    # end
 
-    it 'renders show' do
-      expect(response).to render_template('posts/show')
-    end
+    # it 'renders show' do
+    #   expect(response).to render_template('posts/show')
+    # end
 
-    it 'response body has the right placeholder' do
-      expect(response.body).to include('Post#show')
-    end
+    # it 'response body has the right placeholder' do
+    #   expect(response.body).to include('Post#show')
+    # end
   end
```

Comment:
- [ ] [OPTIONAL] Good job so far! I appreciate what you have done but some of the other tests failed in `./spec/requests/...` spec files, it would be so nice if you solve the problem before the merge of your `PR` 🙏🏼 

![image](https://github.com/Rafiduzzaman/blog-app/assets/63915024/f860f7a4-3cec-4ee1-ad5f-0d31b1c7b4cf)